### PR TITLE
vmspawn: Fix incorrect/broken links in the man page

### PR DESCRIPTION
**Pull request by klausenbusk**

### CODE REVIEW

File: man/systemd-vmspawn.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
   <refsect1>
     <title>Description</title>
     <para><command>systemd-vmspawn</command> may be used to start a virtual machine from an OS image. In many ways it is similar to <citerefentry
-    project='man-pages'><refentrytitle>systemd-nspawn</refentrytitle><manvolnum>1</manvolnum></citerefentry>, but
+    ><refentrytitle>systemd-nspawn</refentrytitle><manvolnum>1</manvolnum></citerefentry>, but
```
Comment (YHNdnzj) [inline man/systemd-vmspawn.xml]: Please move `>` to the previous line.

File: man/systemd-vmspawn.xml [hunk 2]
```diff
@@ -330,7 +330,7 @@
           <term><option>--private-users=</option><replaceable>UID_SHIFT[:UID_RANGE]</replaceable></term>
 
           <listitem><para>Controls user namespacing under <option>--directory=</option>.
-          If enabled, <citerefentry project='man-pages'><refentrytitle>virtiofsd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+          If enabled, <citerefentry project='url'><refentrytitle url='https://manpages.debian.org/unstable/qemu-system-common/virtiofsd.1.en.html'>virtiofsd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
```
Comment (YHNdnzj) [inline man/systemd-vmspawn.xml]: No, we use the Rust version of `virtiofsd`. However, it appears that it doesn't have a man page... So maybe just switch to `<command>virtiofsd</command>`?

**Review by YHNdnzj — CHANGES_REQUESTED** [review]

File: man/systemd-vmspawn.xml (hunk 2 above)
Comment (klausenbusk) [reply man/systemd-vmspawn.xml]: Done. Am I supposed to wrap the line and if yes how? The [`.vimrc`](https://github.com/systemd/systemd/blob/main/.vimrc) states the line width is 109 characters for XML files, but that has not been followed for this file.

Comment (YHNdnzj) [reply man/systemd-vmspawn.xml]: It's not a strict limit, so generally OK if some lines exceed that a bit.

Comment (klausenbusk) [reply man/systemd-vmspawn.xml]: I tried a thing.

**Pull request merged**